$OpenBSD: patch-src_Text_XML_HaXml_Schema_TypeConversion_hs,v 1.1 2014/10/30 09:40:39 dcoppa Exp $

parse errors from Haddock

src/Text/XML/HaXml/Schema/TypeConversion.hs:41:51:
    parse error (possibly incorrect indentation or mismatched brackets)

src/Text/XML/HaXml/Schema/TypeConversion.hs:81:5:
    parse error on input `renameLocals'

--- src/Text/XML/HaXml/Schema/TypeConversion.hs.orig	Tue Oct 28 16:53:24 2014
+++ src/Text/XML/HaXml/Schema/TypeConversion.hs	Wed Oct 29 20:23:44 2014
@@ -38,7 +38,6 @@ typeLift s = s{ schema_items =
                                                   Just (Right
                                                     (ComplexType
                                                        {complex_name=Just t'}))
-                                                  {-| t==t'-}
                                                     -> (e:)
                                                   _ -> id
                   _                          -> id
@@ -79,8 +78,8 @@ typeLift s = s{ schema_items =
 --   previously locally-typed, we will instead assume in the pretty-printer
 --   that it can always replace a missing typename with the element name, and
 --   have it resolve to something sensible.
-    renameLocals :: SchemaItem -> SchemaItem
-    renameLocals s = s
+renameLocals :: SchemaItem -> SchemaItem
+renameLocals s = s
 --  renameLocals (SchemaElement e)
 --                 | Left (NT{theName=n,theType=Nothing}) <- elem_nameOrRef e
 --                 = SchemaElement e{ elem_nameOrRef = Left (NT{theName=n
